src/test/FuzzerImpl.cpp

```diff
@@ -982,7 +982,11 @@ void
 TransactionFuzzer::initialize()
 {
     resetRandomSeed(1);
-    mApp = createTestApplication(mClock, getFuzzConfig(0));
+    auto cfg = getFuzzConfig(0);
+#ifdef BEST_OFFER_DEBUGGING
+    cfg.BEST_OFFER_DEBUGGING_ENABLED = mBestOfferDebuggingEnabled;
```

Comment:
This should always be `true` if `BEST_OFFER_DEBUGGING` is defined, no need to actually support toggling it from the command line.

Reply:
Done.